### PR TITLE
update layout user

### PR DESCRIPTION
**Pull request by truongcr7**
## Related Tickets:
- [#Ticket ID]

## WHAT this PR does?
- Update layout user

## Notes Impacted Areas:
- N/A.

## Attachments:
- 

https://user-images.githubusercontent.com/78576331/133022385-9a4723b5-312a-469a-84e8-fab276747529.mp4

### CODE REVIEW

File: resources/views/user/layouts/layout.blade.php [hunk 1]
```diff
@@ -5,48 +5,81 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <meta name="csrf-token" content="{{ csrf_token() }}">
-    <title>Book Reviewing Admin</title>
+    <title>Book Reviewing System</title>
```
Comment (longnv-99) [inline resources/views/user/layouts/layout.blade.php:8]: i18n @@

Comment (truongcr7) [reply resources/views/user/layouts/layout.blade.php:8]: cả title luôn hả

Comment (longnv-99) [reply resources/views/user/layouts/layout.blade.php:8]: ừ

**Review by hungsondo — APPROVED** [review]

**Comment by truongcr7** [comment]
ready

**Comment by truongcr7** [comment]
ready

**Pull request merged**